resources/zib/zib-TreatmentDirective2.xml

```diff
@@ -290,7 +290,7 @@
     <element id="Consent.provision.actor:agreementParty.reference">
       <path value="Consent.provision.actor.reference" />
       <short value="Patient / Representative / HealthProfessional" />
-      <definition value="* Patient as the person with whom the agreement was made.&#xD;&#xA;* Person who represents the patient in health care matters and with whom the agreement has been made (too).&#xD;&#xA;* HealthProfessional who has made the agreement regarding the treatement directive." />

```

Comment:
Is de spelfout ook gemeld bij het zibcentrum?

Reply:
Ik heb deze fout nu gemeld in ZIB-1921.